CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # bedrock-vue-wallet ChangeLog
 
-## 23.0.0 - 2023-11-xx
+## 24.0.0 - 2023-11-dd
```

Comment:
This should be 23.  Change here, remove the 23 line below, and merge Changed sections.